### PR TITLE
[chore] increase version and update maintainers

### PR DESCRIPTION
**Pull request by felixT2K**
This PR:

- increase version to 0.7.0a0 (next release will be minor and not patch)
- add Olivier and myself  😅 also to the list of maintainers

I will cleanup the issue trackers in the next days 👍🏼 

Any feedback is welcome 🤗

### CODE REVIEW

**Comment by felixdittrich92** [comment]
ok issue trackers are updated (for 0.8.0 we can decide later what we want to keep or maybe move to a later release)

File: pyproject.toml
Comment (odulcy-mindee) [inline pyproject.toml:1]: Haha thanks

File: setup.py
Comment (odulcy-mindee) [inline setup.py:1]: I need to discuss that with the others internally

Comment (felixdittrich92) [reply setup.py:1]: @odulcy-mindee how does it look ? :)

**Review by odulcy-mindee — APPROVED** [review]

**Pull request merged**